src/clientManager.js

```diff
@@ -85,6 +85,8 @@ ClientManager.prototype.loadUser = function(name) {
 			json
 		));
 	}
+
+	return client;
```

Comment:
Did you mean something like this?

```js
	let client = this.findClient(name);
	if (!client) {
		client = new Client(
			this,
			name,
 			json
		));
		this.clients.push(client);
	}
	return client;
```

Otherwise `client` is undefined and ESLint barks.